tensorrt_llm/_torch/visual_gen/models/flux/pipeline_flux2.py

```
@@ -319,6 +319,14 @@ def post_load_weights(self) -> None:
             # Enable TeaCache with FLUX.2-specific polynomial coefficients
             self._setup_teacache(self.transformer, FLUX2_TEACACHE_COEFFICIENTS)
 
+    DEFAULT_GENERATION_PARAMS = {
+        "height": 1024,
+        "width": 1024,
+        "num_inference_steps": 50,
+        "guidance_scale": 3.5,
+        "max_sequence_length": 512,
+    }
```

Ruff (0.15.9)

[warning] Mutable default value for class attribute

(RUF012)